Presentation.Web/app/components/it-advice/it-advice-modal-view.html

```diff
@@ -24,7 +24,7 @@ <h3 class="modal-title pull-left">{{advisName}}</h3>
 
                 <div class="form-group">
                     <label for="toExternal" class="control-label small">Til modtager via email</label>
-                    <input id="toExternal" type="text" data-ng-disabled="!isEditable('ToEmail')" placeholder="Skriv email" class="form-control input-sm" data-ng-model="externalTo" ng-pattern="multipleEmailValidationRegex" data-element-type="adviceEmailToInput">
+                    <input id="toExternal" name="externalEmail" type="text" data-ng-disabled="!isEditable('ToEmail')" placeholder="Skriv email" class="form-control input-sm" data-ng-model="externalTo" ng-pattern="multipleEmailValidationRegex" data-element-type="adviceEmailToInput">
```

Comment:
De her to felter skal ikke have data-element type de har jo et id, så jeg tænker at data-element-type skal væk og så skal testen bruge de id'er der rent faktisk er. Fint med navnet til at tjekke state

Reply:
@JacobMalling jeg tillod mig at rydde op.. no more data-element type i den fil. Der var slet ikke brug for det